pkg/pipelines/config/config.go

```diff
@@ -71,37 +81,23 @@ func (m *Manifest) AddService(envName, appName string, svc *Service) error {
 }
 
 // GetCICDEnvironment returns the CICD Environment if one exists.
-func (m *Manifest) GetCICDEnvironment() (*Environment, error) {
-	envs := []*Environment{}
-	for _, env := range m.Environments {
-		if env.IsCICD {
-			envs = append(envs, env)
+func (m *Manifest) GetCICD() (*Cicd, error) {
```

Comment:
Couple of things here...

I'd rename `Cicd` to `CICDConfig`, or possibly `PipelinesConfig`.

It doesn't need to return an error, if you look at the code, it never returns a non-nil case, but you don't need to, as that was for when you could have multiple environments.